CHANGELOG.md

```diff
@@ -13,3 +13,11 @@
 ## Previous versions
 
 Please check [release/0.25-stable](https://github.com/decidim/decidim/blob/release/0.25-stable/CHANGELOG.md) for previous changes.
+
+#### Base64 images migration
+
+PR [\#8250](https://github.com/decidim/decidim/pull/8250) Replaces the default base64 editor images attachment with the use of ActiveStorage attachments. Also adds a task to parse all editor contents and replace existing base64 images with attachments. The task parses all the attributes which can be edited from admin using the WYSIWYG editor. The task requires an argument with the email of an admin used to create EditorImage instances. To run this task execute:
+
+```
+rails decidim:editor:migrate_inline_images_to_active_storage[admin_email]
```

Comment:
```suggestion
rails decidim:active_storage_migrations:migrate_editor_inline_images_to_active_storage[admin_email]
```

I think this would be better so it's consistent with other active storage migrations tasks that we already have